code/test_data_generator/src/yaml_data.cpp

```diff
@@ -277,6 +277,7 @@ std::string test_data::full_example()
        << "    g: {value: 9.81, unit: m/s^2}\n"
        << "    rho: {value: 1000, unit: kg/m^3}\n"
        << "    nu: {value: 1.18e-6, unit: m^2/s}\n"
+       << "    air rho: {value: 1.225, unit: kg/m^3}\n"
```

Comment:
I agree that using greek letters isn't optimal, but I followed your convention. In formulas I would have written this rho_air (with the greek symbol as 'rho' and 'air' in subscript), which is why I chose `rho_air` in the code. However in the YAML input we are not limited by variable names and we can use spaces, so 'air rho' seemed more human-readable to me (since 'air' qualifies 'rho', as a substitute for 'the rho of air', which English language would reduce to 'air rho'). But it can be changed if you think it is more consistent!